### PR TITLE
READY : Highlight selected and under cursor cells

### PR DESCRIPTION
**Pull request by qarux**

### CODE REVIEW

File: module/game/src/main.rs [hunk 1]
```diff
@@ -393,7 +470,12 @@ fn main()
 
   /* highlighting */
   #[ cfg( not( target_arch = "wasm32" ) ) ]
-  app.add_system( highlight_under_cursor );
+  app.add_system_set
+  (
+    SystemSet::on_update( GameState::GameStart )
+      .with_system( select_cell )
```
Comment (dmvict) [inline module/game/src/main.rs:476]: Please, update formatting.

File: module/game/src/main.rs [hunk 2]
```diff
@@ -300,12 +311,78 @@ fn highlight_under_cursor
...
+    else
+    {
+      Color::rgba( 1.0, 0.0, 0.0, 1.0 )
+    };
+    highlight.highlight( ( x, y ), color );
+  }
+
+  if let Some( pos ) = selected_cell.single().pos
+  {
+    let x = pos.x as u8;
```
Comment (dmvict) [inline module/game/src/main.rs:331]: Reduce number of conversions, please.

File: module/game/src/main.rs [hunk 3]
```diff
@@ -300,12 +311,78 @@ fn highlight_under_cursor
   let camera = q_camera.single();
   let cell = cursor_to_cell( interaction.last_cursor_position, window_size, camera.projection_matrix() );
 
-  if cell.x < 8.0 && cell.y < 8.0 && cell.x >= 0.0 && cell.y >= 0.0
+  if let Some( cell ) = cell
   {
-    highlight.highlight( ( cell.x as u8, cell.y as u8 ), Color::rgba( 1.0, 1.0, 1.0, 0.2 ) );
+    let x = cell.x as u8;
+    let y = cell.y as u8;
+    let color = if game.piece_at( 8 * y + x ) != core::Piece::None
```
Comment (dmvict) [inline module/game/src/main.rs:318]: Please change condition from `!=` to `==`.

**Comment by dmvict** [comment]
Please, resolve conflicts and update code taking into account review comments.

**Comment by qarux** [comment]
All done

**Pull request merged**